### PR TITLE
[UI-29] - Create or integrate component for phone number (with valid country code)

### PR DESCRIPTION
**Pull request by stuniel**
#### Task

- **Issue or task referred:** UI-29
- **Feature or Bugfix:** Feature

I created a component for phone number input which uses Dropdown with custom toggle and button item components. Dropdown is provided with a list of countries with flags and prefixes. When selected, a prefix is added to the input value. When a prefix is typed in the input field it appears as selected in Dropdown component and country flag shows up in Dropdown button.

#### Checklist

- [ ] Required unit tests prepared
- [ ] Documentation prepared
- [ ] External licenses validated (ideally MIT)

### CODE REVIEW

File: packages/phone-number-input/tests/PhoneNumberInput.test.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  })
+
+  it('should highlight selected item', () => {
+    let prefixItem = dropdownItems.findWhere(n => n.contains('+48') && n.hasClass(itemPrefixClsName))
+    prefixItem.simulate('click')
+    const index = wrapper.find('Dropdown').props().defaultHighlightedIndex
+    const prefix = countryList[index].prefix
+    expect(prefix).toEqual('+48')
+  })
+})
```
Comment (PiotrGrzechnik) [inline packages/phone-number-input/tests/PhoneNumberInput.test.js]: You can also add a test that covers 'componentWillReceiveProps'

**Comment by rangoo94** [comment]
Could you improve test coverage?

File: packages/combo-box/README.md [hunk 2]
```diff
@@ -27,46 +27,60 @@ It handles all `div` properties, and additionally:
 
 Property name           | Type                  | Default | Description
 ------------------------|-----------------------|:-------:|--------------------------------
+className               | string                | n/a     | Additional class passed to wrapper
```
Comment (rangoo94) [inline packages/combo-box/README.md]: Hell, it's big :)

Comment (rangoo94) [reply packages/combo-box/README.md]: I am working right now on rebuilding comboboxes to simplify them

File: packages/phone-number-input/src/PhoneNumberInput.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  country: '',
+  prefix: ''
+}
+
+/**
+ * Phone number input.
+ *
+ * @property {object} props
+ * @property {string} [props.className]
+ * @property {string || number} [props.maxHeight]
```
Comment (rangoo94) [inline packages/phone-number-input/src/PhoneNumberInput.js]: `@property {string|number}`

Comment (rangoo94) [reply packages/phone-number-input/src/PhoneNumberInput.js]: You could also describe `state` and `_input`

File: packages/phone-number-input/src/countryList.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import countryData from 'country-telephone-data'
```
Comment (rangoo94) [inline packages/phone-number-input/src/countryList.js]: How is big package emitted? At least `import { allCountries } from 'country-telephone-data'` may be better

**Comment by rangoo94** [comment]
@stuniel, could you try to use combo boxes from https://github.com/PublicInMotionGmbH/ui-kit/pull/79?

**Comment by rangoo94** [comment]
I have created #94 which is using this logic and has new ComboBoxes. Thanks, and sorry, but we need that for the current project.

**Pull request closed without merging**